x/crosschain/keeper/gas_payment.go

```diff
@@ -223,11 +223,13 @@ func (k Keeper) PayGasInERC20AndUpdateCctx(
 	)
 	gasObtained := amounts[2]
 
+	// FIXME: investigate small mismatches between gasObtained and outTxGasFee
+	// https://github.com/zeta-chain/node/issues/1303
 	// check if the final gas received after swap matches the gas fee defined
 	// if not there might be issues with the pool liquidity and it is safer from an accounting perspective to return an error
-	if gasObtained.Cmp(outTxGasFee.BigInt()) != 0 {
-		return cosmoserrors.Wrapf(types.ErrInvalidGasAmount, "gas obtained for burn (%s) not equal to gas fee(%s)", gasObtained, outTxGasFee)
-	}
+	//if gasObtained.Cmp(outTxGasFee.BigInt()) != 0 {
```

Comment:
I propose to make it 
"gasObtained > outTxGasFee" instead.

Reply:
Sounds good, made the change: ec2b399a